smriprep/workflows/anatomical.py

```diff
@@ -515,55 +527,107 @@ def _check_img(img):
     )
     fs_isrunning.inputs.logger = LOGGER
 
-    # 5. Surface reconstruction (--fs-no-reconall not set)
-    surface_recon_wf = init_surface_recon_wf(
-        name="surface_recon_wf", omp_nthreads=omp_nthreads, hires=hires
+    # check for FastSurfer .mgz files and
+    #   touch mri/aseg.auto_noCCseg.label_intensities.txt
+    #   to prevent failure in surfaces.py (temporary fix - DEPRECATED)
+    check_fastsurfer = pe.Node(
+        niu.Function(function=_check_fastsurfer), overwrite=True, name="check_fastsurfer"
     )
-    applyrefined = pe.Node(fsl.ApplyMask(), name="applyrefined")
-    # fmt:off
-    workflow.connect([
-        (inputnode, fs_isrunning, [
-            ('subjects_dir', 'subjects_dir'),
-            ('subject_id', 'subject_id')]),
-        (inputnode, surface_recon_wf, [
-            ('t2w', 'inputnode.t2w'),
-            ('flair', 'inputnode.flair'),
-            ('subject_id', 'inputnode.subject_id')]),
-        (fs_isrunning, surface_recon_wf, [('out', 'inputnode.subjects_dir')]),
-        (anat_validate, surface_recon_wf, [('out_file', 'inputnode.t1w')]),
-        (brain_extraction_wf, surface_recon_wf, [
-            (('outputnode.out_file', _pop), 'inputnode.skullstripped_t1'),
-            ('outputnode.out_segm', 'inputnode.ants_segs'),
-            (('outputnode.bias_corrected', _pop), 'inputnode.corrected_t1')]),
-        (brain_extraction_wf, applyrefined, [
-            (('outputnode.bias_corrected', _pop), 'in_file')]),
-        (surface_recon_wf, applyrefined, [
-            ('outputnode.out_brainmask', 'mask_file')]),
-        (surface_recon_wf, outputnode, [
-            ('outputnode.subjects_dir', 'subjects_dir'),
-            ('outputnode.subject_id', 'subject_id'),
-            ('outputnode.t1w2fsnative_xfm', 't1w2fsnative_xfm'),
-            ('outputnode.fsnative2t1w_xfm', 'fsnative2t1w_xfm'),
-            ('outputnode.surfaces', 'surfaces'),
-            ('outputnode.out_aseg', 't1w_aseg'),
-            ('outputnode.out_aparc', 't1w_aparc')]),
-        (applyrefined, buffernode, [('out_file', 't1w_brain')]),
-        (surface_recon_wf, buffernode, [
-            ('outputnode.out_brainmask', 't1w_mask')]),
-        (surface_recon_wf, anat_reports_wf, [
-            ('outputnode.subject_id', 'inputnode.subject_id'),
-            ('outputnode.subjects_dir', 'inputnode.subjects_dir')]),
-        (surface_recon_wf, anat_derivatives_wf, [
-            ('outputnode.out_aseg', 'inputnode.t1w_fs_aseg'),
-            ('outputnode.out_aparc', 'inputnode.t1w_fs_aparc'),
-        ]),
-        (outputnode, anat_derivatives_wf, [
-            ('t1w2fsnative_xfm', 'inputnode.t1w2fsnative_xfm'),
-            ('fsnative2t1w_xfm', 'inputnode.fsnative2t1w_xfm'),
-            ('surfaces', 'inputnode.surfaces'),
-        ]),
-    ])
-    # fmt:on
+    check_fastsurfer.inputs.logger = LOGGER
+
+    # Select which surface reconstruction workflow based on CLI arguments
+    if freesurfer and (not freesurfer):
```

Comment:
This will always evaluate False.

Reply:
Ah, thanks for catching this typo! I intended this to be `not fastsurfer`